smdebug/tensorflow/keras.py

```diff
@@ -539,7 +539,8 @@ def _save_layer_input_and_outputs(self):
                 if self._is_collection_being_saved_for_step(CollectionKeys.LAYERS)
                 else set()
             )
-            self._save_tensor_to_file(export_name, tensor.numpy(), input_collection)
+            if hasattr(tensor, "numpy"):
```

Comment:
do we have to warn the user if this cannot be saved?

Reply:
Done

Reply:
will this below log then be called for every tensor that fails?

Reply:
The function short circuits after printing the log once for every step.